lib/arkana/models/config.rb

```diff
@@ -36,6 +36,8 @@ class Config
   attr_reader :kotlin_jvm_toolchain_version
   # @returns [boolean]
   attr_reader :is_kotlin_multiplatform_module
+  # @returns [boolean]
+  attr_reader :inference_secret_types
```

Comment:
@cepages could you update this PR to change the name of this variable to `should_infer_types` across the board, please? In some instances the argument/parameter passed had a singular form (`*type`) and here it's in plural (`*types`), so I'd like to also aim for consistency in the plurality as well 😊 

I suggest searching for `inference_secret_type` and replacing it across the board 🙇